### PR TITLE
Add matthewgregory user

### PR DESCRIPTION
**Pull request by mammykins**

### CODE REVIEW

File: modules/users/manifests/matthewgregory.pp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Creates the matthewgregory user
+class users::matthewgregory {
+  govuk_user { 'matthewgregory':
+    fullname => 'Matthew Gregory',
+    email    => 'matthew.gregory@digital.cabinet-office.gov.uk',
+    ssh_key  => 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDFZRIVw76giSwMWRoET/nCjzOiwb+qI6X26zvb0ctbudhlXh/tWBiHnvxbqmMft5oyMIpviFIPmcUxH6fN6PQSdt4ME0XvXbyyXwz8xig/P6Y5it5yFKgqcTMQihDb91Bhp1tXne/kD6UZXQ9qIX8gkE0YJOte1LSNVPxZ3GIm+nOyQBFbad8YrWkubFnY8q7ZoaOHVhabagJIMqoLXGDF5vXPot73xDH40P5pbgy5swpRqW8MDIUpOYyC1BmJvrWl2Ql9upIjLd3PWo5CNXnnSHObFRwCQASAniTb5ur+JtBjok7U8e1SwS0B7tEWqADyHw5xfyBD8n4pyDgTJGZGdV4Kheeqo4g6n7K+Ol9IacXtSLB3gT+i3lZ4XOSuq7AnM3l80r7WYaYBiPhCd8UhY1lACe0JwT0J6Taqt511Eb8mdWjWP/j0M3B7aRYENexQkMzzbykZjicpNCA890WWPql34to7zXQ5D1XFVzFzHDsPieWPscp0CF/OtonKmduszsl0XDJDqlGUf70Fuh80p5FCI6jtK7WDtERs4BW1A13KEEIgCkgL7N90dwgZ0voiBQWqDZ7mMEZw0tk9k2/PBn7eyc7Y+p3dPxck9lecYQ8I/BBpZ72wl5cbQUiyNfz7M+ZF7Fsy5qAeauUpFy3i9Ke7PejzKFeEr1d83IZYqQ== matthew.gregory@digital.cabinet-office.gov.uk
+',
```
Comment (erino) [inline modules/users/manifests/matthewgregory.pp]: The `',` need to be on the line above

Comment (karlbaker02) [reply modules/users/manifests/matthewgregory.pp]: @erino which line are you referring to? This seems to be consistent with other user manifests as it currently is

Comment (mammykins) [reply modules/users/manifests/matthewgregory.pp]: heya, happy to change. I copied someone else's that was approved yesterday, and then find / replaced with my stuff

Comment (erino) [reply modules/users/manifests/matthewgregory.pp]: @karlbaker02 Line 7, I though Puppet would complain if the value on line 6 didn't have a closing quote and a trailing comma

**Pull request merged**